src/apis/core.ts

```diff
@@ -0,0 +1,53 @@
+import axios, { AxiosInstance, AxiosRequestConfig, Method } from 'axios';
+
+const HTTP_METHODS = {
+  GET: 'get',
+  POST: 'post',
+  PUT: 'put',
+  DELETE: 'delete',
+} as const;
+
+const TIME_OUT = 2000;
+
+const axiosInstance = axios.create({
+  baseURL: `${import.meta.env.VITE_API_URL}/${
```

Comment:
```suggestion
  baseURL: `${import.meta.env.VITE_API_URL}:${
```
포트 넘버라면, 실수 한것 같아요~!

Reply:
URL에 포함시키는게 낫겠네요 어차피 별도로 사용하지 않을 것 같아요 ! 그렇게 수정할게요. 감사합니다